src/compas_view2/views/view.py

```diff
@@ -19,23 +19,20 @@ class View(QtWidgets.QOpenGLWidget):
         The view configuration.
     """
 
-    FRONT = 1
-    RIGHT = 2
-    TOP = 3
-    PERSPECTIVE = 4
+    VIEW_ANGLES = {"CUSTOM": 0, "FRONT": 1, "RIGHT": 2, "TOP": 3, "PERSPECTIVE": 4}
```

Comment:
Thanks for the review! 

The reason is for the readability of the config file: the original version would only allow us to put numbers (i,e. `"view_angle":1` ) in the config. While using dict would allow us to indicate the key: `"view_angle": "TOP"`.